docs/working-with-data.md

```diff
@@ -0,0 +1,43 @@
+# Working with Data
+
+The BC Data Service (formerly known as Data BC) encourages and enables the strategic management and sharing of data across the government enterprise and with the public. 
```

Comment:
As far as I know, DataBC is still a service on its own, offered by the BC Data Service Division along with other programs like DIP (Data Innovation Program). Even within our branch (DSS) there are teams working on non-DataBC stuff - e.g., my team (Data Equity Services) and the API team.

Reply:
aha good to know! My intel was false. I will remove this. thanks!